app/views/submission_mailer/confirmation.html.haml

```diff
@@ -0,0 +1,12 @@
+%h3 
+"Szanowna Pani / Szanowny Panie #{@submission.corresponding_author.surname}"
+\
+%p
+informujemy, że przesłane zgłoszenie: 
+= @submission.title 
+zostało przyjęte do systemu. O tym, czy artykuł został zaakceptowany poinformujemy w osobnym mailu. Status zgłoszenia można sprawdzić pod adresem: 
+= @submission.url
```

Comment:
Ten fragment (o statusie) proszę usunąć, bo to jeszcze nie jest na tym etapie zaimplementowane.